### PR TITLE
Term entry html attributes title

### LINKED ISSUE #1529: [Term Entry] HTML Attributes: `title`
### Reason/inspiration (optional)

Please write a new entry under: content/html/concepts/attributes/terms/ about the global attribute `title` in HTML.

A "Syntax" section with pseudo code snippet.
An "Example" section with a HTML code snippet. If an image is added, the file should go in the `media/` folder.

To get started with the PR and entry file:

Under content/html/concepts/attributes/terms/, create a new folder called `title`. Then create a new file called `title.md` under content/html/concepts/attributes/terms/`title`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new term for HTML attributes: title
- Closes #1529 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/attributes/terms/title/title.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - 'Elements'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`title`** global attribute contains and can display the specified information about the element it belongs to.
+
+> **Note:** Although the `title` attribute can display additional information on the screen, it does not work with touch screens, keyboard-only navigation, or assistive technologies.
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/title/title.md:18]: The note about assistive technologies is contradicted by the bullet point about `<iframe>` below

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cslylla I did a first review and only had one comment.

File: content/html/concepts/attributes/terms/title/title.md (hunk 1 above)
Comment (cslylla) [reply content/html/concepts/attributes/terms/title/title.md]: Thank you, I have changed the text to resolve that!

File: content/html/concepts/attributes/terms/title/title.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The `title` attribute can be used with any HTML [`element`](https://www.codecademy.com/resources/docs/html/elements) by adding the attribute to the opening tag. The `text` value of the attribute can be any string, including empty string (`""`) and multi-line text. This `text` will appear next to the element as a tooltip text (hint) while hovering over the element it is used with.
+
+For nested elements, the `title` attribute will be inherited from the parent element if the child element has no `title` attribute.
+
+### Special Use Cases
+
+While any element can have a `title` attribute, it provides a special function for some elements. When the `title` attribute is used with:
+
+- the [`<abbr>`](https://www.codecademy.com/resources/docs/html/elements/abbr) element, the `title` must contain a description or expansion of the abbreviation.
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/title/title.md:34]:
```suggestion
- [`<abbr>`](https://www.codecademy.com/resources/docs/html/elements/abbr): The `title` must contain a description or expansion of the abbreviation.
```

File: content/html/concepts/attributes/terms/title/title.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- the [`<iframe>`](https://www.codecademy.com/resources/docs/html/elements/iframe) element, the `title` should describe the embedded content for assistive technologies.
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/title/title.md:35]:
```suggestion
- [`<iframe>`](https://www.codecademy.com/resources/docs/html/elements/iframe): The `title` should describe the embedded content for assistive technologies.
```

File: content/html/concepts/attributes/terms/title/title.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+- the [`<input>`](https://www.codecademy.com/resources/docs/html/elements/input) element in a form the information is only visible when hovering over the element. It is better to use a `<label>` element that is always visible.
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/title/title.md:36]:
```suggestion
- [`<input>`](https://www.codecademy.com/resources/docs/html/elements/input): This element only displays information when a user is hovering over the element. It is better to use a `<label>` element that is always visible.
```

File: content/html/concepts/attributes/terms/title/title.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+- the `<link rel="stylesheet">` element, the `title` specifies the default and alternate stylesheets.
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/title/title.md:37]:
```suggestion
- `<link rel="stylesheet">`: The `title` specifies the default and alternate stylesheets.
```

File: content/html/concepts/attributes/terms/title/title.md (hunk 2 above)
Comment (caupolicandiaz) [reply content/html/concepts/attributes/terms/title/title.md:34]: I think it's a little cleaner to start with the link name, then colon, description

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla Just a few minor comments. Tag me when you're finished. Thanks

**Comment by cslylla** [comment]
@caupolicandiaz I accepted the proposed changes ☺️

**Review by caupolicandiaz — APPROVED** [review]
@cslylla Looks good.

**Comment by SSwiniarski** [comment]
@cslylla Congrats on your newest entry. The live version can be found here: https://www.codecademy.com/resources/docs/html/attributes/title

image

**Pull request merged**